docs/machine-learning/tutorials/sentiment-analysis-model-builder.md

```diff
@@ -60,63 +54,68 @@ Each row in the *wikipedia-detox-250-line-data.tsv* dataset represents a differe
 1 | == OK! ==  IM GOING TO VANDALIZE WILD ONES WIKI THEN!!!
 0 | I hope this helps.
 
+## Create a Model Builder config file
+
+When first adding a machine learning model to the solution it will prompt you to create an `mbconfig` file. The `mbconfig` file keeps track of everything you do in Model Builder to allow you to reopen the session.
+
+1. In **Solution Explorer**, right-click the *SentimentRazor* project, and select **Add** > **Machine Learning Model...**.
+1. In the dialog, name the Model Builder project **SentimentAnalysis**, and click **Add**.
+
 ## Choose a scenario
 
 ![Model Builder wizard in Visual Studio](../media/model-builder-scenarios.png)
 
 To train your model, you need to select from the list of available machine learning scenarios provided by Model Builder.
 
-1. In **Solution Explorer**, right-click the *SentimentRazor* project, and select **Add** > **Machine Learning Model...**.
```

Comment:
` Machine Learning Model....` it should be ` Machine Learning Model (ML.NET)`

Reply:
This is what I see on the latest version.

![image](https://user-images.githubusercontent.com/46974588/200689028-63a9d0f3-57a6-40dd-9795-6b0646277192.png)